src/routes/api/userRoutes.js

```diff
@@ -1,9 +1,18 @@
 import express from 'express'
+import { UserController } from '../../controllers/userController'
 
 const route = express.Router()
 
 route.get('/', (req, res, next) => {
-    res.status(200).json({ status: 200, message: "this will return all users", data: "" })
+    new UserController().getAllUsers(req, res, next)
+})
```

Comment:
Replace it with `route.get("/", UserController.getAllUsers)`. Implement this for the other routes